zebrad/Cargo.toml

```diff
@@ -109,12 +109,8 @@ error-debug = ["color-eyre/track-caller", "color-eyre/capture-spantrace"]
 #
 # Release builds are defined as "cfg(not(debug_assertions))".
 # https://docs.rs/tracing/latest/tracing/level_filters/index.html#compile-time-filters
-release_max_level_warn = ["tracing/release_max_level_warn", "log/release_max_level_warn"]
 release_max_level_info = ["tracing/release_max_level_info", "log/release_max_level_info"]
-release_max_level_debug = ["tracing/release_max_level_debug", "log/release_max_level_debug"]
 
-max_level_warn = ["tracing/max_level_warn", "log/max_level_warn"]
-max_level_info = ["tracing/max_level_info", "log/max_level_info"]
```

Comment:
log 0.4.22 introduced a breaking change where no multiple levels can be defined. https://github.com/rust-lang/log/pull/627

They might revert the decision as there are complains: https://github.com/rust-lang/log/issues/635 but for now it seems we need to remove some stuff. I am not sure what could happen by doing this change, input from others is appreciated.

Reply:
Let's keep these features, but have them only enable features in `tracing` and not `log`? 

We're not using `log` directly anyway, I'm not sure it'll affect anything except the `LOG_STATIC_MAX_LEVEL` logged in the tracing component's `new()` and `reload_filter()` methods.

If possible, it might be better to set the max static level with a compilation config variable later so `--all-features` will work, these levels should be mutually exclusive, so compilation features probably aren't the right way to do that.

Reply:
Thanks! I think keep the features for tracing is the right thing to do for now. I did that at https://github.com/ZcashFoundation/zebra/pull/8664/commits/94c9b3e9c2ea427fb1d59efaa57c6705327742fe